src/graphql/schema/collection/createCollectionGqlType.ts

```diff
@@ -53,7 +53,7 @@ export default function createCollectionGqlType<TValue> (
               description: field.description,
```

Comment:
Huh, good catch. It seems the TypeScript compiler (at least the version I'm using) is hella forgiving. It's also really REALLY hard to figure out where syntax has gone wrong with TypeScript as compared to Babel which is so much more helpful with its errors.